app/models/form/account_batch.rb

```diff
@@ -25,19 +26,21 @@ def save
       suppress_follow_recommendation!
     when 'unsuppress_follow_recommendation'
       unsuppress_follow_recommendation!
+    when 'suspend'
+      suspend!
     end
   end
 
   private
 
   def follow!
-    accounts.find_each do |target_account|
+    accounts.each do |target_account|
```

Comment:
Why this change? Especially since `approve!` is changed from a `each` to a `find_each`.

Reply:
Realistically there is no difference, the account_ids array will never be > 1000 accounts for this to make a difference, but CodeClimate complains about using each. I don't know why it didn't complain here. I originally changed all find_each to each, and this apparently got left behind.